ci/module-kmm-ci-build.yaml

```diff
@@ -8,9 +8,13 @@ spec:
     container:
       modprobe:
         moduleName: kmm_ci_a
-      imagePullPolicy: Never
       kernelMappings:
         - regexp: '^.+$'
           containerImage: image-registry.openshift-image-registry.svc:5000/default/kmm-kmod:$KERNEL_FULL_VERSION
+          registryTLS:
+            insecure: true
```

Comment:
We do not need this.

Reply:
This is just `git` misleading here. Since `module-kmm-ci-build.template.yaml` was renamed to `module-kmm-ci-build.yaml` git shows the diff from the `module-kmm-ci-crd` (probably because it is the closes to this one after the change).

So the `registryTLS` was already present in the module (this PR didn't change it). The only differences in this file (compared to its previous version `module-kmm-ci-build.template.yaml` are:
* removing the build args
* removing the build secret (which was added back later on and will be pull m/s at some point)

Reply:
If you want to validate the change you can check the diff between `module-kmm-ci-build.yaml` in this PR and `module-kmm-ci-build.template.yaml` from the `main` branch.

Reply:
Let me make it simple for you. Here is the real diff:
```
root kernel-module-management-midstream (e2e-cleanup) $ sdiff ci/module-kmm-ci-build.yaml ../module-kmm-ci-build.template.yaml
---                                                             ---
apiVersion: kmm.sigs.x-k8s.io/v1beta1                           apiVersion: kmm.sigs.x-k8s.io/v1beta1
kind: Module                                                    kind: Module
metadata:                                                       metadata:
  name: kmm-ci                                                    name: kmm-ci
spec:                                                           spec:
  moduleLoader:                                                   moduleLoader:
    container:                                                      container:
      modprobe:                                                       modprobe:
        moduleName: kmm_ci_a                                            moduleName: kmm_ci_a
      kernelMappings:                                                 kernelMappings:
        - regexp: '^.+$'                                      |         - literal: KVER_CHANGEME
          containerImage: image-registry.openshift-image-regi |           containerImage: image-registry.openshift-image-regi
          registryTLS:                                                    registryTLS:
            insecure: true                                                  insecure: true
          build:                                                          build:
                                                              >             buildArgs:
                                                              >               - name: CI_BUILD_ARG
                                                              >                 value: some-build-arg
                                                              >             secrets:
                                                              >               - name: build-secret
            dockerfileConfigMap:                                            dockerfileConfigMap:
              name: kmm-kmod-dockerfile                                       name: kmm-kmod-dockerfile
  selector:                                                       selector:
    task: kmm-ci                                                    task: kmm-ci

```

Reply:
Let's anyway remove that line? 🙂

Reply:
It means that we will skip TLS u/s but not m/s. Is this what we want?

Reply:
Yes. Upstream, we use a plain HTTP registry. Downstream, we have TLS enabled and all the required CAs.

Reply:
Ack. Removed